### ISSUE TITLE
Manager can't find bin-64 folder if gw2 is installed on SSD where windows is not installed on.

### ISSUE THREAD

**Issue by CodeMasterWille** [fmmmlee/GW2-Addon-Manager#82]

**Comment by Seyhzade** [comment]
yeah, i have same issue to. İ formatted my pc and changed from m2 ssd to ssd. Now i cant install Add On manager. Keep giving "Cant find bin64/ or /bin64/ error. No matter what i did, couldnt fix it.

**Comment by Seyhzade** [comment]
Found the problem. İf u accidently instal gw2 x64 in x86 Program files, u gettin that error. I re instaled in x64 program files now its working

**Comment by CodeMasterWille** [comment]
I did manage to fix it by putting the addon manager folder in my gw2 folder on my ssd!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
